lib/order.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/send_sms'
+
+class Order
+
+  attr_reader :menu
+
+  def initialize
+    @order = []
+  end
+
+  def add_menu(menu)
```

Comment:
If I understand correctly, you have this method so that you can have access to the dishes. Could you not initialise the class with a menu and inject the dependency there? Given that you can't have an order without a menu, it would save you from having to call add_menu.